src/operator/image/image_random-inl.h

```diff
@@ -79,16 +79,15 @@ inline bool ToTensorType(const nnvm::NodeAttrs& attrs,
 
 // Operator Implementation
 
-template<int req>
 struct totensor_forward {
   template<typename DType>
   MSHADOW_XINLINE static void Map(uint32_t c, float* out_data, const DType* in_data,
                                   const int length, const int channel, const int step,
                                   const float normalize_factor = 255.0f) {
       #pragma omp parallel for
       for (int i = 0; i < length; ++i) {
-        KERNEL_ASSIGN(out_data[step + c*length + i], req,
-                      (in_data[step + i*channel + c]) / normalize_factor);
+        out_data[step + c*length + i] =
```

Comment:
you should take care of the req rather than disable the check, this is dangerous for any behavior our of your usecase right now